tests/handler/test_invalid_usage.py

```diff
@@ -34,7 +34,7 @@ class OperationHandlerOverridesNameInconsistentlyWithServiceDefinition(_TestCase
     def build():
         @nexusrpc.service
         class SD:
-            my_op: nexusrpc.Operation[None, None]  # type: ignore[reportUninitializedInstanceVariable]
+            my_op: nexusrpc.Operation[None, None]
 
         @service_handler(service=SD)
         class SH:
```

Comment:
Are these `reportUnusedClass` and `reportUnusedParameter` issues valid? If not, we should consider disabling them for test due to noise.

Reply:
I'm still thinking about that. I'm tempted to make the codebase compliant, e.g. using underscore-prefixed variable names where they are intentionall not used.